bower.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "n-image",
-  "version": "0.0.0",
```

Comment:
@ironsidevsquincy the version number in bower.json is completely optional, bower only really uses git tags.